app/src/DemoResetTask.php

```diff
@@ -23,11 +23,11 @@ public function run($request)
 
         echo "Resetting database...\n<br>";
         echo "<pre>";
-        echo htmlentities(`nice -n 5 cat .reset/db_reset.sql | $mysqlBin -h $CLI_server -u $CLI_dbuser $CLI_dbpassArg -D $CLI_database &> /dev/stdout`);
+        echo htmlentities(`nice -n 5 cat .reset/db_reset.sql | $mysqlBin -h $CLI_server -u $CLI_dbuser $CLI_dbpassArg -D $CLI_database &> /dev/stdout` || '');
```

Comment:
This command doesn't return any output which comes back as a `null` response instead of a blank string.